tests/polytope_test.py

```diff
@@ -48,5 +67,84 @@ def comparison_test():
         np.array([1, 0, 0])))
 
 
+def region_rotation_test():
+    p = pc.Region([pc.Polytope(A, b)])
+    p1 = pc.Region([pc.Polytope(A, b)])
+    p2 = pc.Region([pc.Polytope(Ab2[:, 0:2], Ab2[:, 2])])
+    p3 = pc.Region([pc.Polytope(Ab3[:, 0:2], Ab3[:, 2])])
+    p4 = pc.Region([pc.Polytope(Ab4[:, 0:2], Ab4[:, 2])])
+
+    p = p.rotation(0, 1, np.pi/2)
+    print(p.bounding_box)
+    assert(p == p2)
```

Comment:
I'm not going to change this because all of the other tests use `assert` as a function. They can all be changed together in some other pull request.